RestaurantManagment/src/main/java/com/flmfoods/dto/ItemResponseDto.java

```diff
@@ -0,0 +1,25 @@
+package com.flmfoods.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class ItemResponseDto {
+
+	
```

Comment:
```suggestion
	private long itemId;
```
Add itemId in response